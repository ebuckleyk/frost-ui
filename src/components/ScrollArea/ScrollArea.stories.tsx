import * as React from 'react';
import { Meta, StoryObj } from '@storybook/react';
import { ScrollArea } from './ScrollArea';

type ComponentType = React.ComponentProps<typeof ScrollArea>;
const meta: Meta<ComponentType> = {
  component: ScrollArea,
};

export default meta;

type Story = StoryObj<ComponentType>;
export const Default: Story = {};
