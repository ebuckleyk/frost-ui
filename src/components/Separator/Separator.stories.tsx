import * as React from 'react';
import { Meta, StoryObj } from '@storybook/react';

import { Card } from '../Card';
import { Separator } from './Separator';

function SeparatorDemo() {
  return (
    <Card className="p-5">
      <div className="space-y-1">
        <h4 className="text-sm font-medium leading-none">Radix Primitives</h4>
        <p className="text-sm text-muted-foreground">An open-source UI component library.</p>
      </div>
      <Separator className="my-4" />
      <div className="flex h-5 items-center space-x-4 text-sm">
        <div>Blog</div>
        <Separator orientation="vertical" />
        <div>Docs</div>
        <Separator orientation="vertical" />
        <div>Source</div>
      </div>
    </Card>
  );
}

type ComponentType = React.ComponentProps<typeof Separator>;
const meta: Meta<ComponentType> = {
  component: Separator,
  render: SeparatorDemo,
};

export default meta;

type Story = StoryObj<ComponentType>;
export const Demo: Story = {};
