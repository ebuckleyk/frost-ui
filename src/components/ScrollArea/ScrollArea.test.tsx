import * as React from 'react';
import { render } from '@testing-library/react';

import { Separator } from '../Separator';
import { ScrollArea } from './ScrollArea';

const tags = Array.from({ length: 50 }).map((_, i, a) => `v1.2.0-beta.${a.length - i}`);

function Component() {
  return (
    <ScrollArea className="h-72 w-48 rounded-md border">
      <div className="p-4">
        <h4 className="mb-4 text-sm font-medium leading-none">Tags</h4>
        {tags.map((tag) => (
          <>
            <div key={tag} className="text-sm">
              {tag}
            </div>
            <Separator className="my-2" />
          </>
        ))}
      </div>
    </ScrollArea>
  );
}

describe('ScrollArea', () => {
  it('should render ScrollArea component to match snapshot', () => {
    const result = render(<Component />);
    expect(result).toMatchSnapshot();
  });
});
