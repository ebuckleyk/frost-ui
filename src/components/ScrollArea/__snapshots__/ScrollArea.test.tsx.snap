// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ScrollArea > should render ScrollArea component to match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="relative overflow-hidden h-72 w-48 rounded-md border"
        dir="ltr"
        style="position: relative; --radix-scroll-area-corner-width: 0px; --radix-scroll-area-corner-height: 0px;"
      >
        <style>
          [data-radix-scroll-area-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-scroll-area-viewport]::-webkit-scrollbar{display:none}
        </style>
        <div
          class="h-full w-full rounded-[inherit]"
          data-radix-scroll-area-viewport=""
          style="overflow-x: hidden; overflow-y: scroll;"
        >
          <div
            style="min-width: 100%; display: table;"
          >
            <div
              class="p-4"
            >
              <h4
                class="mb-4 text-sm font-medium leading-none"
              >
                Tags
              </h4>
              <div
                class="text-sm"
              >
                v1.2.0-beta.50
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.49
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.48
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.47
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.46
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.45
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.44
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.43
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.42
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.41
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.40
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.39
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.38
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.37
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.36
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.35
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.34
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.33
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.32
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.31
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.30
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.29
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.28
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.27
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.26
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.25
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.24
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.23
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.22
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.21
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.20
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.19
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.18
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.17
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.16
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.15
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.14
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.13
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.12
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.11
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.10
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.9
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.8
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.7
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.6
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.5
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.4
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.3
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.2
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
              <div
                class="text-sm"
              >
                v1.2.0-beta.1
              </div>
              <div
                class="shrink-0 bg-border h-[1px] w-full my-2"
                data-orientation="horizontal"
                role="none"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="relative overflow-hidden h-72 w-48 rounded-md border"
      dir="ltr"
      style="position: relative; --radix-scroll-area-corner-width: 0px; --radix-scroll-area-corner-height: 0px;"
    >
      <style>
        [data-radix-scroll-area-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-scroll-area-viewport]::-webkit-scrollbar{display:none}
      </style>
      <div
        class="h-full w-full rounded-[inherit]"
        data-radix-scroll-area-viewport=""
        style="overflow-x: hidden; overflow-y: scroll;"
      >
        <div
          style="min-width: 100%; display: table;"
        >
          <div
            class="p-4"
          >
            <h4
              class="mb-4 text-sm font-medium leading-none"
            >
              Tags
            </h4>
            <div
              class="text-sm"
            >
              v1.2.0-beta.50
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.49
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.48
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.47
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.46
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.45
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.44
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.43
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.42
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.41
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.40
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.39
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.38
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.37
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.36
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.35
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.34
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.33
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.32
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.31
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.30
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.29
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.28
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.27
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.26
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.25
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.24
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.23
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.22
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.21
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.20
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.19
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.18
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.17
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.16
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.15
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.14
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.13
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.12
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.11
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.10
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.9
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.8
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.7
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.6
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.5
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.4
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.3
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.2
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
            <div
              class="text-sm"
            >
              v1.2.0-beta.1
            </div>
            <div
              class="shrink-0 bg-border h-[1px] w-full my-2"
              data-orientation="horizontal"
              role="none"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
